oozie-to-airflow/mappers/file_archive_mappers.py

```diff
@@ -84,24 +86,24 @@ def add_file(self, oozie_file_path: str) -> None:
         )
 
 
-# pylint: disable=too-few-public-methods
-class ArchiveMixin:
-    """Mixin for archive node"""
+class ArchiveMapper:
+    """ Converts an archive oozie node """
```

Comment:
This description is misleading. This code not only converts one node.  It also searches for nodes. In next step, it performs a conversion of all nodes.

Reply:
I've changed it in the current version to `Extracts all archive paths from an Oozie node`. Similarly for file mapper.